### ISSUE TITLE
Create a discard function when a C# discard assign (_) is converted to __ in VB

### ISSUE THREAD

**Issue by elGuille-info** [paul1956/CSharpToVB#53]
I note that the discard assignement in C# _ (underscore) is converted to __ (double underscore)
But there is no double underscore function or property defined in the VB code so... an Error is thrown.

If it's possible the converter shoul add a property like this:
````vb
' Shared is not optional only if is using a Class instead of a Module
Private [Shared] Property __ As Object
````

I don't know how to add this functionality in your code... but hope it's help, and it's easy for you to do ;-)

Thanks!
Guillermo

**Comment by paul1956** [comment]
@elGuille-info  great idea I will look into it.

**Comment by paul1956** [comment]
@elGuille-info found multiple errors one that has been in converter since it first supported Tuples. It will create a WriteOnly property when needed. All the logic was already there to support other helper functions in Modules and Classes.

Fixed in 5.0.0.18/5.0.0.10
